webdriver-spec.html

```diff
@@ -3946,6 +3946,9 @@
  <li><p>Let <var>centre point</var> be a pair of <var>rect</var>’s
   (<a>rectangle x coordinate</a> + <var>horizontal centre offset</var>)
   and (<a>rectangle y coordinate</a> + <var>vertical centre offset</var>)
+  
+ <li><p>If <a>command</a> is intercepted by an <var>element</var> with a higher 
+  z-index, an <a>element intercepting click</a> <a>error</a> is returned.
```

Comment:
I think this conversation has gotten a bit out of hand when putting it in the perspective of what I'm actually doing here. I'm not creating a new end point or a new way to detect whether an element click has been intercepted, I am merely wanting to change the output error of what happens when an element click is supposed to succeed after passing all the checks in https://w3c.github.io/webdriver/webdriver-spec.html#element-click but before point 14 it fails because another element was in the way. At the moment, it is a very unhelpful "unknown error" which cannot be caught specifically. 

I do not know the ins and outs of the stacking order and element visibility/presence algorithms and am not sure why I'm being asked all these questions about them. I don't mean to be snappy or brusque, and I do want to get this change through, I just don't know the answers to half these questions and I'm quite confused as to why there is a much larger discussion around this than I'd thought initially. 

Cheers,
James